express-post-json/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const app = express();
+let nextId = 1;
+const grades = {};
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  const { name, course, score } = req.body;
+  if (!name || !course || !score) {
+    res.status(400).send('Missing argument(s)');
+    return;
+  }
+  grades[nextId] = { id: nextId, name, course, score };
+  res.status(201).send(grades[nextId++]);
+});
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
```

Comment:
You were supposed to send back an array of grades like in the previous exercise

Reply:
sorry I didn't catch that I updated it to respond with an array now